### ISSUE TITLE
[Bug]: Could not get unknown property 'kotlinVersion' when building

### ISSUE THREAD

**Issue by pwfcurry** [braze-inc/braze-expo-plugin#6]
TLDR @braze/expo-plugin v0.4.0 isn't compatible with react-native-appboy-sdk v1.39 & v1.40 -- downgrade to v1.38 to fix

### Which Platforms?

Android

### Which React Native Version?

0.68.2

### Which @braze/expo-plugin version?

0.4.0

### Which react-native-appboy-sdk SDK version?

1.40.0

### Repro Rate

100% of the time

### Steps To Reproduce

```
yarn add @braze/expo-plugin@0.4.0 react-native-appboy-sdk@1.40.0
# add braze config to app.json…
eas build --platform android --local
```

### Expected Behavior

Build succeeds

### Actual Incorrect Behavior

Build fails

### Verbose Logs

```
[RUN_GRADLEW] Using expo modules
…
[RUN_GRADLEW]   - braze-expo-plugin (0.4.0)
…
[RUN_GRADLEW] FAILURE: Build failed with an exception.
[RUN_GRADLEW] * Where:
[RUN_GRADLEW] Build file '/private/var/…/build/node_modules/react-native-appboy-sdk/android/build.gradle' line: 33
[RUN_GRADLEW] * What went wrong:
[RUN_GRADLEW] A problem occurred evaluating project ':react-native-appboy-sdk'.
[RUN_GRADLEW] > Could not get unknown property 'kotlinVersion' for object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.
[RUN_GRADLEW] * Try:
[RUN_GRADLEW] > Run with --stacktrace option to get the stack trace.
[RUN_GRADLEW] > Run with --info or --debug option to get more log output.
[RUN_GRADLEW] > Run with --scan to get full insights.
[RUN_GRADLEW] * Get more help at https://help.gradle.org
[RUN_GRADLEW] BUILD FAILED
[RUN_GRADLEW] Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.
[RUN_GRADLEW] You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.
[RUN_GRADLEW] See https://docs.gradle.org/7.3.3/userguide/command_line_interface.html#sec:command_line_warnings
[RUN_GRADLEW] 6 actionable tasks: 6 executed
[RUN_GRADLEW] in 55s
[RUN_GRADLEW] Error: Gradle build failed with unknown error. See logs for the "Run gradlew" phase for more information.
```

### Additional Information

Due to https://github.com/Appboy/appboy-react-sdk/pull/164 (released in 1.39.0), we've been unable to build our app:

```
> EAS_LOCAL_BUILD_SKIP_CLEANUP=1 eas build --platform android --local --non-interactive
…
[RUN_GRADLEW] A problem occurred evaluating project ':react-native-appboy-sdk'.
[RUN_GRADLEW] > Could not get unknown property 'kotlinVersion' for object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.
```

Downgrading to v1.38.0 fixes the issue.

**Comment by radixdev** [comment]
Hi @pwfcurry ,

We're investigating the issue and should have a fix/resolution soon. Apologies for the delay.

**Comment by radixdev** [comment]
Hi @pwfcurry ,

We've identified a fix in the react repo and will update you once that fix is live in this expo repo.

**Comment by pwfcurry** [comment]
Thanks @wesleyorbin @radixdev!
